Developing-new-features-with-feature-gating.md

```diff
@@ -145,6 +145,91 @@ spyOnProperty(platformFeatureService, 'status', 'get').and.returnValue(
 );
 ```
 
+#### Backend
+
+For example, if you wish to write a test with the required feature flag enabled, you can do so by following the steps below (the approach to testing with the feature flag disabled is the same up to step 4):
+
+1. Import the following modules into the test file:
+
+```python
+from core.domain import platform_feature_services as feature_services # (alias is optional)
+from core.domain import platform_parameter_domain
+from core.domain import platform_parameter_list
+from core.domain import platform_parameter_registry as registry # (alias is optional)
+```
+
+2. Clear the parameter registry and create a mock feature flag (using the actual enum of the feature flag -- we recommend you do it in the `setUp` hook), like so:
+
+```python
```

Comment:
@nithinrdy For consistency with the codebase, could you please use 4-space indentation in all Python code blocks?

Reply:
Done.